.neo/docker/neo-cli/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:stable-slim
+
+# Install the apt-get packages
+RUN apt-get update
+RUN apt-get install -y libicu-dev libleveldb-dev screen
```

Comment:
No, not current. The point of the `docker` image. It allows you build off the it; build your own `docker` image.

### Example
This is one way how it could be used.
```Dockerfile
FROM ghcr.io/cschuchardt88/neo-cli:latest

ENTRYPOINT ["screen", "-dmS","node","neo-cli"]
```

Reply:
Also install location is at `/opt/neo-cli` directory.

Reply:
`neo-cli` we build doesn't require `dotnet` on the host machine. The binary `neo-cli` requires `libicu-dev` to run without `dotnet` installed.